experience_interests.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
+<html lang="en-us">
+<head>
+  <meta charset="utf-8">
+  <link href='http://fonts.googleapis.com/css?family=Lato:300,700,300italic,700italic' rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" type="text/css" href="css/styles.css">
+  <title>Experience and Interests</title>
+</head>
+
+<body>
+
+  <nav>
+    <a href="index.html"> About Me </a> 
+    <a href="experience_interests.html"> Experience </a>
+  </nav>
+
+  <h1>EXPERIENCE</h1>
+
+
+  <img class="primary" src="images/leslie-knope-waffles.jpg" alt="default profile image" title="profile image"/> </img>
+
+  <h3>An Accomplished Pawnee Citizen</h3>
+
+   <p>Leslie is a proud member of numerous action committees in and around City Hall, including, but not limited to: the Equal Opportunity Committee, the Fun in the Sun Committee, the Clean Restroom Task Force, the Increased Access to Restrooms Task Force, the Handicapped Restroom Task Force, the Committee for Repainting Handicapped Parking Spaces Task Force and the Task Force to Reduce the Number of Public Restrooms.</p>
+
+   <p>Leslie graduated in the top 5% of her class at Pawnee North High School. She was Co-Vice President of the student body and a member of Model United Nations, Key Club, Debate Club, Mock Trial, Young Republicans, Young Democrats, Young Independents (founding member), Drama Club, Historical Society, Orchestra, Color Guard and junior varsity field hockey.</p>
+
+  <img src="images/waffles.jpg"> <img src="images/paperwork.jpg"> <img src="images/anneperkins.jpg"> 
```

Comment:
@chloeeffron The image cannot be seen. Here you are requesting ` <img src="images/waffles.jpg">` , but you named the image `waffles.JPG`. HTML is case sensitive, in the future try to keep all of your files in lowercase 🔨